xonsh/xoreutils/uname.py

```diff
@@ -0,0 +1,161 @@
+import platform
+import sys
+from xonsh.xoreutils.util import arg_handler
+
+__version__ = '0.1'
```

Comment:
I don't think this makes sense to version separately from xonsh.

Reply:
that is because the original uname have --version what is it suppose to return ?